snuba/subscriptions/codecs.py

```diff
@@ -2,20 +2,27 @@
 
 from arroyo.backends.kafka import KafkaPayload
 
+from snuba.datasets.entities.factory import ENTITY_NAME_LOOKUP
+from snuba.datasets.entity import Entity
 from snuba.query.exceptions import InvalidQueryException
 from snuba.subscriptions.data import (
     DelegateSubscriptionData,
     InvalidSubscriptionError,
     LegacySubscriptionData,
     SnQLSubscriptionData,
     SubscriptionData,
-    SubscriptionType,
 )
+from snuba.subscriptions.entity_subscription import SubscriptionType
 from snuba.subscriptions.worker import SubscriptionTaskResult
 from snuba.utils.codecs import Codec, Encoder
 
 
 class SubscriptionDataCodec(Codec[bytes, SubscriptionData]):
+    dataset_name: str
```

Comment:
Was this left over from a previous implementation? I don't think it's needed anymore

Reply:
oops missed that